### DISCUSSION TITLE
GREAT WORK! - v1.9.9 is seriously awesome!

### DISCUSSION THREAD

**Discussion by jbrown1597** [AndreyPavlenko/Fermata#425] [General]
This thing blows every other AA app out of the water. Everything is working smooth as butter!

- The "launcher" style screen that allows you to add / remove apps is **AMAZING** 🌟
- Addition of ChatGPT was a _really_ nice touch
- YouTube & browser seemingly working great!
- Screen mirroring touch implementation is _far_ superior to that of Screen2Auto (**respectfully**)

To the devs - thank you so much 💘

**Edit**: P.S - I'm actively using the app & will be for the forseeable future - if you need any assistance / testing / suggestions, I'd be happy to help!

**Comment by maripic69** [comment]
Когато инсталирах версия 1.9.9на АА ми излиза Fermata Media Player, но пак няма достъп до You Tube и браузер!!!Какво трябва да направя!!

**Comment by ersansaper** [comment]
Fermata 1.9.9 sürümünü yükledim, Android Auto görünmüyor

**Comment by SOFO888** [comment]
Thank you Andrey. This version is awesome. The app bar is really a great improvement.
